pom.xml

```diff
@@ -377,6 +378,14 @@
                 <version>1.0.0</version>
             </dependency>
             
+            <dependency>
+                <groupId>pl.edu.icm.spark-utils</groupId>
+                <artifactId>spark-utils</artifactId>
+                <version>0.0.1-SNAPSHOT</version>
+            </dependency>
```

Comment:
Yes, I thought about it a little... I am not sure if it's a problem now, because we have explicitely defined the scope of these depenedencies as provided (in iis) and the project scope will take precendence.